arelle/plugin/validate/NL/rules/nl_kvk.py

```diff
@@ -188,3 +188,32 @@ def rule_nl_kvk_3_1_3_2 (
             msg=_('xbrli:scenario must only contain content defined in XBRL Dimensions specification.'),
             modelObject = contextsWithImproperContent
         )
+
+
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[
+        DISCLOSURE_SYSTEM_INLINE_NT19
+    ],
+)
+def rule_nl_kvk_3_2_1_1 (
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    NL-KVK.3.2.1.1: precision should not be used on numeric facts.
+    """
+    factsWithPrecision = []
+    for facts in val.modelXbrl.factsByQname.values():
```

Comment:
Rather than grabbing these as a list of lists, can we not just use `modelXbrl.facts`, which is a list of all the facts?